src/mixins/controls.js

```diff
@@ -173,14 +200,12 @@ const OrbitControlsMixin = (superclass) => class extends superclass {
         this.render();
     }
 
-    showSecondAxes() {
-        const secondAxes = [this.scene2.x, this.scene2.y, this.scene2.z].filter(x => x);  // assert no `undefined`;
-        this.scene2.add(...secondAxes);
-    }
 
     hideSecondAxes() {
-        const secondAxes = [this.scene2.x, this.scene2.y, this.scene2.z].filter(x => x);  // assert no `undefined`;
-        this.scene2.remove(...secondAxes);
+        this.scene2.remove(this.scene2.secondAxesGroup);
```

Comment:
We should keep `showSecondAxes` function, and move some of the code from `enableOrbitControls` into it